employee-rostering-frontend/src/store/store.ts

```diff
@@ -39,7 +39,7 @@ export interface StoreConfig {
 
 export function configureStore(
   { restBaseURL }: StoreConfig,
-  preloadedState?: AppState,
+  preloadedState?: Partial<AppState>,
```

Comment:
Should be `DeepPartial<AppState>`, I guess.

Reply:
Nope; that introduces bugs; the deep partials are not merged with the initial state set in the reducer

Reply:
Are you sure? Based on https://github.com/reduxjs/redux/issues/433#issuecomment-220123880 it should work when using `combineReducers()`.

Reply:
But the simple example from the comment doesn't actually use deep partials... so that doesn't prove it works in our case.

Reply:
I tried it at first, and end up with undefined, so I switch to using Partials.